ui/components/ui/box/box.d.ts

```diff
@@ -290,7 +290,7 @@ export interface BoxProps extends React.HTMLAttributes<HTMLElement> {
    * Use TEXT_ALIGN const from '../../../helpers/constants/design-system';
    * Accepts responsive props in the form of an array.
    */
-  textAlign?: BoxTextAlign | BoxTextAlignArray;
+  textAlign?: BoxTextAlign | BoxTextAlignArray | TextAlign;
```

Comment:
Ahh I think Box still uses the deprecated version of TextAlign. Would you be able to update it to use the `TextAlign` enum and remove instances of `TEXT_ALIGN` Similar to other enums in the file 

Create new type
```
export type TextAlignArray = [
  TextAlign,
  TextAlign?,
  TextAlign?,
  TextAlign?,
];
```

Then you should be able to remove the old types

```
export type BoxTextAlign = (typeof TEXT_ALIGN)[keyof typeof TEXT_ALIGN] | null;
export type BoxTextAlignArray = [
  BoxTextAlign,
  BoxTextAlign?,
  BoxTextAlign?,
  BoxTextAlign?,
];
```

```suggestion
  textAlign?: TextAlign | TextAlignArray;
```